lesson_10/test_tasks.py

```diff
@@ -0,0 +1,82 @@
+"""Module test tasks"""
+
+
+from homework_10 import log_event
+import logging
+from datetime import date
+from venv import logger
+
+"""Check if two arguments were provided"""
+
+def test_two_strings_provided():
+
+    try:
+        func_loging = log_event("tester")
+    except TypeError as miss_type:
+        print("Missing positional argument")
+
+
+#test_two_strings_provided()
+
+#Test Data
+name = "test"
+name_1 = "test1"
+status_s = "success"
+status_e = "expired"
+status_f = "failed"
+
+info = logging.info("20")
+warn = logging.warning("30")
+err = logging.error("40")
+
+logger_data = open('login_system.log', 'r')
+data = logger_data.read()
+
+"""Could find only one level of logs"""
+
+def test_logger_level():
+    get_logger_code = logger.getEffectiveLevel()
+
+    if get_logger_code == 30 or 20:
+        print("Warning or Info")
+    else:
+        print("Error")
```

Comment:
Checks withing test ( here and below) we do with assert statement
No sense to add if-else